### PR TITLE
Add 1.5.0 release notes

### PR DESCRIPTION
**Pull request by kuenishi**

### CODE REVIEW

**Comment by kuenishi** [comment]
We're still using org.

File: RELEASE-NOTES.org [hunk 1]
```diff
@@ -1,3 +1,9 @@
+* Stanchion 1.5.0 Release Notes
+** Addisions
+- Update to lager 2.0.3
+** Bugs Fixed
+- Include lager_syslog to release package (#81)
+- MP initiated objects remains after delete/create buckets riak_cs/#475 fixed by riak_cs/#857 stanchon/#78
```
Comment (shino) [inline RELEASE-NOTES.org]: It seems to me that MP is too abbreviated.

**Comment by shino** [comment]
Very tiny issue but user related: https://github.com/basho/stanchion/pull/79

**Comment by shino** [comment]
:+1: after above two comments are addressed.

**Comment by kuenishi** [comment]
I added a line about Erlang version, like riak_cs.

**Comment by shino** [comment]
:+1: :sparkles:

**Pull request merged**